supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -182,10 +182,10 @@ Admonitions should draw the reader’s attention to certain information. Keep ad
 
 Valid admonition types:
 
-- NOTE
-- IMPORTANT
-- WARNING
-- TIP
+NOTE:: Additional information, such as product performance, configuration, or support. Ignoring this information does not impact product usage.
```

Comment:
I'm not sure if "Ignoring this information does not impact product usage" is always necessarily going to be true. I'd suggest to remove this sentence.